@hubless/issues/templates/tasks.md

```
@@ -0,0 +1,50 @@
+![[docs/components/issues/status-summary.md]]
+
+## Task Dependency Graph
+
+- Keep task dependencies in-sync with the JSON files.
+- A future automation will render the DAG and feed the frontier worker pool.
+
+## How to Update
+
+1. Add or edit a JSON file under `@hubless/issues/tasks/` (see `../schema/task.schema.json`).
+2. Do not edit generated tables. Update the JSON under `@hubless/issues/tasks/` and re-run `make docs`.
+3. Once a task is `DONE` and has badges **Tested**, **Documented**, **Shipped**, move the JSON file to `tasks.archive/` and record it in `tasks.archive.md`.
```

markdownlint-cli2 (0.18.1)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)